README.md

```diff
@@ -45,424 +45,7 @@ Linux
-```js
-var plugin = {
-    author: 'YourHandle',
-    version: 1.0,
-    name: 'Sample JavaScript Plugin',
-
-    onEventAsync: function (gameEvent, server) {
-    },
-
-    onLoadAsync: function (manager) {
-    },
-
-    onUnloadAsync: function () {
-    },
-
-    onTickAsync: function (server) {
-    }
-};
-```
-#### Required Properties
-- `author` &mdash; [string] Author of the plugin (usually your name or online name/alias)
-- `version` &mdash; [float] Version number of your plugin (useful if you release several different versions)
-- `name` &mdash; [string] Name of your plugin (be descriptive!)
-- `onEventAsync` &mdash; [function] Handler executed when an event occurs
-    - `gameEvent` &mdash; [parameter object] Object containing event type, origin, target, and other info (see the GameEvent class declaration)
-    - `server` &mdash; [parameter object] Object containing information and methods about the server the event occured on (see the Server class declaration)  
-- `onLoadAsync` &mdash; [function] Handler executed when the plugin is loaded by code
-    - `manager` &mdash; [parameter object] Object reference to the application manager (see the IManager interface definition) 
-- `onUnloadAsync` &mdash; [function] Handler executed when the plugin is unloaded by code (see live reloading)
-- `onTickAsync` &mdash; [function] Handler executed approximately once per second by code *(unimplemented as of version 2.\*)*
-    - `server` &mdash; [parameter object] Object containing information and methods about the server the event occured on (see the Server class declaration) 
-### Live Reloading
-Thanks to JavaScript's flexibility and parsability, the plugin importer scans the plugins folder and reloads the JavaScript plugins on demand as they're modified. This allows faster development/testing/debugging.
-
----
-### Misc
-#### Anti-cheat
-This is an [IW4x](https://iw4xcachep26muba.onion.link/) only feature (wider game support planned), that uses analytics to detect aimbots and aim-assist tools.  
-To utilize anti-cheat, enable it during setup **and** copy `_customcallbacks.gsc` from `userraw` into your `IW4x Server\userraw\scripts` folder.  
-
-The anti-cheat feature is a work in progress and as such will be constantly tweaked and may not be 100% accurate, however the goal is to deter as many cheaters as possible from IW4x.
-#### Database Storage
-By default, all **IW4MAdmin** information is stored in `Database.db`. 
-
-Should you need to reset your database, this file can simply be deleted.  
-Additionally, this file should be preserved during updates to retain client information.   
-
-Setting the `ConnectionString` and `DatabaseProvider` properties in `IW4MAdminSettings.json`  
-will allow **IW4MAdmin** to use alternate methods for database storage
+## [More information may be found in our wiki.](https://github.com/RaidMax/IW4M-Admin/wiki)
```

Comment:
Let's change this verbiage to "Explore the wiki (linked) to find more information"